test/world_cup_test.rb

```diff
@@ -0,0 +1,78 @@
+require './lib/world_cup'
+require './lib/team'
+require './lib/player'
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'pry'
+
+class WorldCupTest < Minitest::Test
+
+  def test_it_exists
+    france = Team.new("France")
+    croatia = Team.new("Croatia")
+    world_cup = WorldCup.new(2018, [france, croatia])
+    assert_instance_of WorldCup, world_cup
+  end
+
+  def test_it_finds_all_active_players_in_world_cup_with_position
+    france = Team.new("France")
+    mbappe = Player.new("Kylian Mbappe", :forward)
+    griezmann = Player.new("Antoine Griezmann", :forward)
+    pogba = Player.new("Paul Pogba", :midfielder)
+    france.add_player(mbappe)
+    france.add_player(griezmann)
+    france.add_player(pogba)
+    croatia = Team.new("Croatia")
+    modric = Player.new("Luka Modric", :midfielder)
+    perisic = Player.new("Ivan Perisic", :forward)
+    vida = Player.new("Domagoj Vida", :defender)
+    croatia.add_player(modric)
+    croatia.add_player(perisic)
+    croatia.add_player(vida)
+    world_cup = WorldCup.new(2018, [france, croatia])
+    expected = [mbappe, griezmann, perisic]
+    actual = world_cup.active_players_by_position(:forward)
+    assert_equal expected, actual
+  end
+
+  def test_active_players_in_world_cup_found_with_only_teams_non_eliminated
+    france = Team.new("France")
+    mbappe = Player.new("Kylian Mbappe", :forward)
+    griezmann = Player.new("Antoine Griezmann", :forward)
+    pogba = Player.new("Paul Pogba", :midfielder)
+    france.add_player(mbappe)
+    france.add_player(griezmann)
+    france.add_player(pogba)
+    croatia = Team.new("Croatia")
+    modric = Player.new("Luka Modric", :midfielder)
+    perisic = Player.new("Ivan Perisic", :forward)
+    vida = Player.new("Domagoj Vida", :defender)
+    croatia.add_player(modric)
+    croatia.add_player(perisic)
+    croatia.add_player(vida)
+    world_cup = WorldCup.new(2018, [france, croatia])
+    croatia.eliminated = true
+    expected = [mbappe, griezmann]
+    actual = world_cup.active_players_by_position(:forward)
+    assert_equal expected, actual
+  end
+
+  def test_all_players
```

Comment:
I would like to see an assertion in this test - even if it is not passing! (maybe especially if it is not passing 😆 )